README.md

```diff
@@ -1 +1,160 @@
-# webpack-todo
\ No newline at end of file
+# Portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Images\html_css.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Webpack</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [WEBPACK PRJECT] <a name="about-project"></a>
+
+**[Abiy Capstone One]** This is a webpack project created during my Microverse course. 
```

Comment:
- [Optional ] Kindly I Recommend you to change  **[Abiy Capstone One]** to the current project activity.